docs/section6/Rule6-8.md

```diff
@@ -37,16 +37,20 @@
           - Get normalized space lighting schedule for B_RMR: `normalized_schedule_b = normalize_space_schedules(space_b.interior_lighting)`  
 
         - Get normalized space lighting schedule: `normalized_schedule_p = normalize_space_schedules(space_p.interior_lighting)`
+        
+        - Check if lighting uses schedule to model occupancy control, get occupancy control credit: `if interior_lighting.are_schedules_used_for_modeling_occupancy_control: control_credit = data_lookup(table_G3_7, space.lighting_space_type, interior_lighting.occupancy_control_type)`
```

Comment:
I think the logic should be that if "are_schedules_used_for_modeling_occupancy_control"=true or null then proceed with logic; and if are_schedules_used_for_modeling_occupancy_control"=true then produce undetermined outcome.

Reply:
Sorry, I didnt follow the logic here. Could you explain it again?